pkg/util/ip.go

```diff
@@ -42,7 +46,34 @@ func ToCommonString(ip string, prefix uint32) string {
 	return fmt.Sprintf("%s/%d", ip, prefix)
 }
 
-func GetValidAddressCount(cidr string) int {
+func GetAddressRange(addr string) (net.IP, net.IP, *net.IPNet, error) {
```

Comment:
I think we can save a lot of code if we refer to the implementation of MetalLB
https://sourcegraph.com/github.com/metallb/metallb@b6678ebb6cb7b0ec591e88de9b34808d59212b1e/-/blob/internal/config/config.go#L422:6

For example: convert IP range to CIDR

Reply:
OK, I can try